### PR TITLE
Cobbduceus 1.2/X; Sanguibital is replaced with Helbital, A C2 Favoring Near-Death Experiences!

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
## About The Pull Request

Sangui's C/B ratio is not good and the mechanic makes it hardly (never) used so I've just removed it.

### Helbital

Helbital is a C2 brute (bital) drug that scales based off your other base damages. However, this damage must be maintained or it will become toxic (until you're as close to death as you were when first taking the drug, but now all the damage is toxin).

Advanced doctors can work around this side effect by healing the toxin/burn and replacing with oxy damage which is easier to heal post-metabolization. You can also slightly outrace the damage near the end.

It uses the same reaction as sanguibital so don't worry about having to remember a new recipe.

There's also a VERY SMALL percentage to find yourself playing rock paper scissors with spirits in a quest for full revival. Not CL'd since it's too rare to be practical (and some people like surprise-mechanics 😏).

## Why It's Good For The Game

Sangui removal is because it wasn't ever used and the entire gimmick made it undesirable.

Hopefully this chem will be more enticing, especially for advanced doctors.

## Changelog
:cl:
del: Sanguibital is now gone
add: Helbital has replaced sanguibital (same recipe)! Helbital is a C2 drug that heals brute damage based on how damaged you are in burn/tox/oxy. The downside is that the other damages must be maintained or the drug becomes toxic instead.
/:cl:

### CODE REVIEW

**Comment by zxaber** [comment]
>Burns, toxins, and asphyxition will increase healing but these damages
>asphyxiation

So if someone comes in with a bruise, inject them with Helbital and strangle them until they get better.

**Comment by fluffe9911** [comment]
The beatings will continue until health improves.

**Comment by necromanceranne** [comment]
I can't believe the doctor is going to choke me out to heal me UwU

**Comment by ExcessiveUseOfCobblestone** [comment]
> So if someone comes in with a bruise, inject them with Helbital and strangle them until they get better.

choke a b*tch to heal

**Comment by ExcessiveUseOfCobblestone** [comment]
mfw you think the doctor is healing you with helbital but then you see the proboscis

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -5,30 +5,72 @@
...
+	. = TRUE
+	var/cccombo = M.getToxLoss() + M.getOxyLoss() + M.getFireLoss()
+	var/healed_this_iteration = FALSE
+	if(cccombo >= beginning_combo)
+		M.adjustBruteLoss(FLOOR(cccombo/-30,0.1)) //every 3 damage adds 0.1 per tick
+		healed_this_iteration = TRUE
+	else
+		M.adjustToxLoss((beginning_combo-cccombo)*0.1) //If you are just healing instead of converting the damage we'll KINDLY do it for you AND make it the most difficult!
+
+	if(healed_this_iteration && !reaping && prob(0.0001)) //janken with the grim reaper!
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: why not bump this hel bartering thing up to a 0.1% chance, but move the check for it from on_mob_life() to on_mob_add()?

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 2]
```diff
@@ -5,30 +5,72 @@
...
+		reaping = TRUE
+		var/list/RockPaperScissors = list("rock" = "paper", "paper" = "scissors", "scissors" = "rock") //choice = loses to
+		helbent = M.apply_status_effect(/datum/status_effect/necropolis_curse,CURSE_BLINDING)
+		to_chat(M, "<span class='hierophant'>Malevolent spirits appear before you, barting your life in a 'friendly' game of rock, paper, scissors. Which do you choose?</span>")
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: "barting"

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 3]
```diff
@@ -5,30 +5,72 @@
...
+			to_chat(M, "<span class='hierophant'>You tie, and the malevolent spirits disappear... for now.</span>")
+			reaping = FALSE
+		else if(RockPaperScissors[RPSchoice] == grim) //You lost!
+			to_chat(M, "<span class='hierophant'>You lose, and the malevolent spirits smirk eerily as they surround your body.</span>")
+			M.dust()
+			return
+		else //VICTORY ROYALE
+			to_chat(M, "<span class='hierophant'>You win, and the malevolent spirits fade away as well as your wounds.</span>")
+			SSmedals.UnlockMedal(MEDAL_HELBITALJANKEN,M.client)
+			M.revive(TRUE)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: will this or the neutral effect clear the necropolis curse of blinding?

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]:
```suggestion
		to_chat(M, "<span class='hierophant'>Malevolent spirits appear before you, bartering your life in a 'friendly' game of rock, paper, scissors. Which do you choose?</span>")
```

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: no it's not meant to be practical in any sense IE plasmarifle in vending machine

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: intended for this guy since it fullheals.

neutral i'm willing to live with, it's 1 in a million.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: look YOU COULD HAVE DONE THIS

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 4]
```diff
@@ -5,30 +5,72 @@
...
+/datum/reagent/medicine/C2/helbital/on_mob_metabolize(mob/living/carbon/M)
+	beginning_combo = M.getToxLoss() + M.getOxyLoss() + M.getFireLoss() //This DOES mean you can cure Tox/Oxy and then do burn to maintain the brute healing that way.
+	return ..()
+
+/datum/reagent/medicine/C2/helbital/on_mob_life(mob/living/carbon/M)
+	. = TRUE
+	var/cccombo = M.getToxLoss() + M.getOxyLoss() + M.getFireLoss()
+	var/healed_this_iteration = FALSE
+	if(cccombo >= beginning_combo)
+		M.adjustBruteLoss(FLOOR(cccombo/-30,0.1)) //every 3 damage adds 0.1 per tick
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: if you don't make this a much better ratio, this chem will never get used

how does this interact with that blood drunk eye item that multiplies your max health and current damage by 10?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: this is free healing if the user is still taking damage of any source besides brute. 

You can also play around the chem as people have joked about.

**Comment by Time-Green** [comment]
This chem allows creative fuckery 👍

**Comment by FancyCyclops** [comment]
This chem allows creative _fuck_ery.

**Comment by Fhaxaris** [comment]
Almost no one in gonna use this, medical on high pop is filled with people who just 
1) open container
2) take out medicine
3) click on patient
otherwise pretty cool, will be very nice

**Comment by Shadowflame909** [comment]
Well. I guess the chemist testing grief mixes on me is now okay

**Comment by ExcessiveUseOfCobblestone** [comment]
If someone is poisoned AND heavily brute'd I fail to see why you WOULDN'T use this. It seems like a nobrainer.

**Comment by Time-Green** [comment]
Most it could theoretically heal is 6-7 per tick and only when in deep crit. Assuming you have heavy brute already you can’t get more than 4. At that point you just use salicyclic acid for more consistent healing. 

In it’s current state it’s probably most useful for healing brute under 25 when someone has very heavy damage from other sources. It definitely has its niche, but it’s alot less exciting than I originally thought. 

I’d say up the healing so that it could be used to heal when brute has a bigger factor, like 50 brute and 120 burn. It would still be gimmicky, but would be a better chem for preparing patients for the treatment of the other damages. 

Or not, it’s a C2 chem and shouldn’t be too useful.

**Comment by ExcessiveUseOfCobblestone** [comment]
you can use both

**Comment by Krysonism** [comment]
I actually miss sanguibital because i think brute -> blood/blood loss is a interesting mechanic and i think it can be salvaged if numbers are adjusted and we explain that you should use it sparingly or have blood replacements nearby.

As for this chem it seems pretty cool

**Comment by Tlaltecuhtli** [comment]
i dont understand what this chemical does

**Comment by Time-Green** [comment]
It heals 1 brute damage for every 30 damage of non-brute damage. If you heal any of the non-brute damage with thus chem, it instead converts the brute damage to toxin damage

**Comment by Tlaltecuhtli** [comment]
on tick?
time to use slimes as leeches and get free brute regen

**Comment by ExcessiveUseOfCobblestone** [comment]
> It heals 1 brute damage for every 30 damage of non-brute damage.

Yep unless it's punishing you for a net loss in non-brute damage from when it was injected.

If the iteration heals you get a one in a million chance to play Rock Paper Scissors / Janken with the grim reaper. Rad! 

>  If you heal any of the non-brute damage with this chem, it instead converts the brute damage to toxin damage

If you heal any of the non-brute damage without replacing it (IE healing 20 tox is ok if you deal 20 oxy/burn; You can freely deal 20 tox then heal that 20 tox though), it INSTEAD returns you to this point with the hardest of the 3 damages (toxin).

Basically:

T/O/F Net **Loss** From Metab Start = Toxin Damage until you get back to Metab Start T/O/F
T/O/F Net **0** From Metab Start = Heals Brute at (30 T/O/F damage : 1 brute per tick)
T/O/F Net **Gain** From Metab Start  = Heals Brute at (30 T/O/F damage : 1 brute per tick)

**Comment by ExcessiveUseOfCobblestone** [comment]
slimes do clone so just step out in space

**Comment by Yttriums** [comment]
Cobby please make an infographic about all of your new chems and post it to discord. I refuse to learn the names and tiers and recipes from the chem list on the wiki or git posts.

**Comment by Tlaltecuhtli** [comment]
can i space assistants and tell them this is how you heal a toolbox in the head?

**Comment by ExcessiveUseOfCobblestone** [comment]
This is what I used to do with cryo and salbu since cryo healed better chilly and salbu lolnoped any oxygen damage.

You still need to be mindful of the other 3 damage types, otherwise yeah that's a valid tactic.

**Comment by Shadowflame909** [comment]
hopefully this becomes the brute Cryo.

My comment about letting the chemist grief me wasn't a downside. lol. It was just an interesting possibility that might negate burn-out of griefchem.

Chemistry's really weird in that sense. As its only two functions is being an NPC medicine maker, and being a griefing tool of mass destruction and shittery

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 5]
```diff
@@ -5,30 +5,73 @@
...
-	color = "#ECEC8D" // rgb: 236	236	141
-	taste_description = "whatever vampires would eat"
+/datum/reagent/medicine/C2/helbital //only REALLY a C2 if you heal the other damages but not being able to outright heal the other guys is close enough to damaging
+	name = "Helbital"
+	description = "Named after the norse goddess Hel, this medicine heals the patient's bruises the closer they are to death. Burns, toxins, and asphyxition will increase healing but these damages must be maintained while the drug is being metabolized or the drug will react negatively."
+	color = "#9400D3"
+	taste_description = "cold and lifeless"
 	overdose_threshold = 35
 	reagent_state = SOLID
+	var/datum/status_effect/necropolis_curse/helbent
```
Comment (nemvar) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: Why exactly is this a variable? Couldn't this just be a boolean?

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 6]
```diff
@@ -5,30 +5,73 @@
...
 	..()
-	return TRUE
+	return
 
-/datum/reagent/medicine/C2/sanguibital/overdose_process(mob/living/carbon/M)
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		H.bleed(2)
+/datum/reagent/medicine/C2/helbital/overdose_process(mob/living/carbon/M)
+	helbent = M.apply_necropolis_curse(CURSE_WASTING | CURSE_BLINDING)
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: it does in this pr

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 5 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: I initially thought you could have multiple sources of status effects; Yes

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: not that it'll matter considering above

[the remark this answers is not in the document]
Comment (nemvar) [reply code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: Welp didn't notice that.

**Comment by Potato-Masher** [comment]
I'm not gonna say anything here other than it's going to be difficult to explain to the Captain why I have to choke them out heal them.

**Comment by necromanceranne** [comment]
@Potato-Masher not if you're a catgirl. :3

**Comment by Shadowflame909** [comment]
Hopefully, ai's don't flip the fuck out over this.

Rule Changes that synchronize with code-changes more when.

>Ai calls sec cause you're strangling the captain

"AI IM A MEDICAL DOCTOR THIS IS HOW I HEAL THEM STOP BEING SO PARANOID"

**Comment by ExcessiveUseOfCobblestone** [comment]
consensual harm is ok for sillycons

**Comment by fluffe9911** [comment]
isnt that only for asimov +?

**Comment by Fhaxaris** [comment]
no i believe a policy change was made recently that made it so the new chems that heal and damage at the same time are ok if they consent. not for other things though.

**Comment by ExcessiveUseOfCobblestone** [comment]
consensual harm has always been okayed IE rage cages and such. If you see a doctor choking someone and the patient lets you know it's okay then you shouldn't stop it before or after this PR. Check silly policy in the rules.

This chem does not require you to harm an individual to heal (it can work if they're already damaged) so I don't expect asimov AIs to start giving damage to amplify the healing. They shouldn't.

**Comment by Shadowflame909** [comment]
It's interesting for sure, but are you worried this'll be more like medicinal armor that you were hoping to remove?

**Comment by ExcessiveUseOfCobblestone** [comment]
you have to undergo at least 45 tox/oxy/burn (almost half your non-crit HP) to get 3 ticks of just brute healing so not particularly. Combo damagers such as the bee sword aren't particularly common anyways.

At best it acts as partial armor for brute but only if you are doing another damage type on top of brute.

**Comment by ExcessiveUseOfCobblestone** [comment]
I think I got a notif last night that respii said this was good, remind me to revert.
